Found 1 known vulnerability, ignored 1 in 1 package
Name          Version ID             Fix Versions
------------- ------- -------------- ------------
python-dotenv 1.2.1   CVE-2026-28684 1.2.2
Attempt 1 failed. Retrying in 10s...
Found 1 known vulnerability, ignored 1 in 1 package
Name          Version ID             Fix Versions
------------- ------- -------------- ------------
python-dotenv 1.2.1   CVE-2026-28684 1.2.2
Attempt 2 failed. Retrying in 10s...
Found 1 known vulnerability, ignored 1 in 1 package
Name          Version ID             Fix Versions
------------- ------- -------------- ------------
python-dotenv 1.2.1   CVE-2026-28684 1.2.2
Attempt 3 failed. Retrying in 10s...
Audit failed after 3 attempts.
Error: Process completed with exit code 1.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -54,7 +54,7 @@ openpyxl = { version = "^3.1", optional = true }
 pydantic = "^2.11.3"
 httpx = "^0.28.1"
 tenacity = "^9.1.2"
-python-dotenv = "^1.1.0"
+python-dotenv = ">=1.2.2"
 typer = {extras = ["all"], version = "^0.15.2"}
 python-json-logger = "^2.0.7"
 urllib3 = "^2.6.3"

